proto/gz/msgs/camerasensor.proto

```diff
@@ -24,21 +24,88 @@ option java_outer_classname = "CameraSensorProtos";
 /// \interface CameraSensor
 /// \brief Information about a camera sensor element
 
-import "gz/msgs/vector2d.proto";
 import "gz/msgs/distortion.proto";
+import "gz/msgs/double.proto";
 import "gz/msgs/header.proto";
+import "gz/msgs/lens.proto";
+import "gz/msgs/sensor_noise.proto";
+import "gz/msgs/vector2d.proto";
 
 message CameraSensor
 {
+  /// \brief Bounding box types.
+  enum BoundingBoxType
+  {
+    /// \brief No bounding box.
+    NO_BOUNDING_BOX = 0;
+
+    /// \brief 2D box that shows the full box of occluded objects
+    FULL_BOX_2D     = 1;
+
+    /// \brief 2D box that shows the visible part of the occluded object
+    VISIBLE_BOX_2D  = 2;
+
+    /// \brief 3D oriented box
+    BOX_3D          = 3;
+  }
+
+  /// \brief Segmentation types.
+  enum SegmentationType
+  {
+    /// \brief No segmentation.
+    NO_SEGMENTATION = 0;
+
+    /// \brief Pixels of same label from different items
+    /// have the same color & id.
+    SEMANTIC        = 1;
+
+    /// \brief Pixels of same label from different items, have different
+    /// color & id. 1 channel for label id & 2 channels for instance id
+    PANTOPIC        = 2;
```

Comment:
```suggestion
    PANOPTIC        = 2;
```

Reply:
[0eb0afa](https://github.com/gazebosim/gz-msgs/pull/201/commits/0eb0afa70d367484b20f4f653bb6b4b9d65e3d9b)

Reply:
It's still wrong. That's a tough word